cg/store/api/findbusinessdata.py

```diff
@@ -317,6 +318,20 @@ def pool(self, pool_id: int) -> models.Pool:
         """Fetch a pool."""
         return self.Pool.get(pool_id)
 
+    def ready_made_library_expected_reads(self, case_id: str) -> float:
+        """Return the target reads of a ready made library case"""
+        case_obj: models.Family = self.family(case_id)
+        application_version = self.ApplicationVersion.query.filter(

```

Comment:
Could you separate the query and the filtering in different functions, something like in this PR: https://github.com/Clinical-Genomics/cg/pull/1545/files. It will make it a lot easier to read, maintain and test (in unit tests)

Reply:
No, but I found a cleaner way to do it! 👍